kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkTaskConfig.java

```diff
@@ -112,6 +112,16 @@
           + " and enable timestamp partitioning for each table. Leave this configuration blank,"
           + " to enable ingestion time partitioning for each table.";
 
+  public static final String BIGQUERY_PARTITION_EXPIRATION_CONFIG = "partitionExpirationMs";
+  private static final ConfigDef.Type BIGQUERY_PARTITION_EXPIRATION_TYPE = ConfigDef.Type.LONG;
+  private static final String BIGQUERY_PARTITION_EXPIRATION_DEFAULT = null;
+  private static final ConfigDef.Importance BIGQUERY_PARTITION_EXPIRATION_IMPORTANCE = ConfigDef.Importance.LOW;
+  private static final String BIGQUERY_PARTITION_EXPIRATION_DOC =
+          "The amount of time, in milliseconds, after which partitions should be deleted from the table. If this " +
+                  "field is set, all data in partitions in this table older than the specified partition " +
+                  "expiration time will be permanently deleted. " +
+                  "Existing tables will not be altered to use this partition expiration.";
```

Comment:
Sorry, I still think the language here needs work. Two issues:

1. The connector can write to several tables, so "the table" and "this table" aren't really accurate here.
2. Like I suggested before, we should be explicit that this only applies to tables the connector creates. The "Existing tables..." notice at the end is helpful, but we can and should be more explicit too.

Here's my suggestion:

```suggestion
          "The amount of time, in milliseconds, after which partitions should be deleted from tables created by the connector. " +
                  "If this field is set, all data in partitions older than the specified partition " +
                  "expiration time will be permanently deleted. " +
                  "Existing tables will not be altered to use this partition expiration.";
```